README.md

```diff
@@ -129,20 +129,76 @@ The above script run only when config files contains latest keys. In order to up
 * In VDI, Paste the keys in credentials file: ~/.aws/credentials (remove the first copied line, looks somethings like: [198477955030_AdministratorAccess])
 * add [default] in first line
 
-The config and credentials file should look like (with own keys): 
+The config and credentials file should look like (with own keys):
 
-'''
+```
 [default]
 export AWS_ACCESS_KEY_ID="ASIAS4NRVH7LD2RRGSFB"
 export AWS_SECRET_ACCESS_KEY="rpI/dxzQWhCul8ZHd18n1VW1FWjc0LxoKeGO50oM"
 export AWS_SESSION_TOKEN="IQoJb3JpZ2luX2VjEGkaCWV1LXdlc3QtMiJH"
-'''
+```
 
 ## Loading data on s3 bucket from VDI (using boto3)
 
 to upload the file(s) on S3 use: /aws-scripts/s3_file_upload.py
 to upload the directory(s) on S3 use: /aws-scripts/s3_bulk_data_upload.py
 
+## AWS Elactic container repository
+
+Following instructions are for creating image repo on ECR and uploading container image
+
+* SSH EC2 contianer and clone PyPRECIS repository:
+
+```
+sudo yum install -y git
+git init
+```
+
+* On VDI repo branch, run the following command:
+
+```
+git push <ec2 host name>:~
+```
+
+* Now checkout the branch on EC2: git checkout develop
+* Install docker
+
+```
+sudo amazon-linux-extras install docker
+```
+
+* build docker image:
+
+```
+sudo docker build .
+```
+
+* goto AWS ECR console and "create repository", make it private and name it
+
+* Once created, press "push commands"
+
+* copy the command and run it on EC2 instance, it will push the container image on record
```

Comment:
do you mean copy each command in turn and run it on the EC2 instance? (there are 
Also I had a permission denied issue when trying to run the `aws ecr get-login-password ` command. `Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run ...`
To fix this I had to run `sudo chmod 666 /var/run/docker.sock`

Reply:
yes .. copy each command and run on ec2

use sudo before 'docker' in the command